contained-innersource.md

```diff
@@ -0,0 +1,70 @@
+## Title  
+
+Contained InnerSource
+
+## Patlet
+
+Apply InnerSource methods to facilitate collaboration in a cross divisional 
+project but don't invest in soliciting contributions from outside of that
+project.
+
+## Problem  
+
+Traditional development practices do not work as effectively for cross
+divisional projects as do InnerSource style practices. However, Management
+does not support adopting InnerSource practices because it regards efforts for
+soliciting and facilitating contributions from outside of the project as
+detrimental for the required efficiency. 
+
+## Context
+
+* The company plans to develop a new software intensive product; multiple
+  business units are involved in the development of this product
+* There is not one single BU which has the required resources or knowledge
+  to do the development on its own.
+* An InnerSource program and required tools exist. 
+* The InnerSource program advocates using the full spectrum of InnerSource
+  practices (_Full Scale InnerSource_).
+
+## Forces  
+
+* The product's importance to company revenue and the committed feature content
+  and dates require a development paradigm that provides known, stable
+  development resources (headcount).
+* _Full scale InnerSource_ includes substantial effort for soliciting and 
+   managing contributions as well as onboarding new contributors, which is 
+   hard to plan for and my have some effect on the ability to control the 
+   timeline for completion and deployment.
+
```

Comment:
* Code visibility/project transaparency is orthogonal to this particular Pattern. Leaving the code visible (the way other InnerSource projects would be) leaves the door open for future, more extensive collaboration opportunities, but the project may view this as possibly distracting their limited resources if unsolicited contributions are submitted. Keeping the code closed to participating members of the project doesn't activate the _Full scale InnerSource_ benefit of possibly higher quality due to having more eyes on the code but might satisfy potential project needs for secrecy (e.g., there may be legal or contractual obligations for the company to fulfill).

Reply:
> the project may view this as possibly distracting their limited resources if unsolicited contributions are submitted.

Perhaps the best solutoin then is to propose a small experiment. For instance, leave the repo visible but don't be overly concerned with soliciting contributions. If it becomes a distraction, it can easily be locked down, if contributions turn out to be useful then the team can start to implement small steps to encourage further contribution.